tox.ini

```diff
@@ -10,7 +10,9 @@ python =
        3.11: py311
 
 [testenv]
-commands = ./runtests.py --nolint
+commands =
+       ./runtests.py --nolint
+       mypy rest_framework_nested
```

Comment:
If possible, please let `mypy` be another check command, that can fail separately from the runtests one.